parts/k8s/addons/azure-cloud-provider.yaml

```diff
@@ -102,7 +102,7 @@ volumeBindingMode: WaitForFirstConsumer
 allowedTopologies:
 - matchLabelExpressions:
   - key: topology.disk.csi.azure.com/zone
-    values: {{GetZones}}
+    values: {{GetAgentPoolZones}}
```

Comment:
We should use above `{{ - if HasAgentPoolAvailabilityZones }}` right?

Reply:
yep

Reply:
fixed